0028-web-monetization/0028-web-monetization.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Web Monetization
-draft: 5
+draft: 6
```

Comment:
Doesn't seem worth it

Reply:
It's an auto-mated check, the effort to change that to figure out what's editorial and what's significant is definitely not worth it 😄